homeassistant/bootstrap.py

```diff
@@ -93,6 +93,11 @@
 from .util.logging import async_activate_log_queue_handler
 from .util.package import async_get_user_site, is_virtual_env
 
+with contextlib.suppress(ImportError):
+    # Ensure anyio backend is imported to avoid it being imported in the event loop
+    from anyio._backends import _asyncio  # noqa: F401
```

Comment:
My first attempt was to try to make the issue to the httpx helper load this, but that means every integration would have to change because it would need to call a new Coro.  Chances are good we load anyway, and we have it marked as a core dep, so seem to be the least risky option.